packages/components/src/index.ts

```diff
@@ -7,6 +7,14 @@ export * from "@kaizen/brand"
 export * from "@kaizen/brand-moment"
 export * from "@kaizen/button"
 export * from "@kaizen/component-base"
+export {
+  Icon,
+  Box,
+  Dropdown,
+  Layout,
+  Spacing,
+  Text,
```

Comment:
Do we want all of these even though most are deprecated patterns? Especially `Text`, which is replaced by `@kaizen/typography`

Reply:
HMMMM this is a tough one. On one hand I want everyone to be able to adopt, but on the other, it would take quite a number of majors to delete them...

Okay lets leave out deprecated ones haha